doc/administrator/administrate.rst

```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
 * ``Roles``: Restricted to the following roles.
 * ``Read/write``: Allows the linked users to change the objects.
 * ``Area``: Active in the following area, if not defined it is active everywhere.
+
+Users
+-----
+
+Each user may have from 1 to n roles, but each user has a default role from
+which are taken the some settings. Currently the default role (defined through
+the "Settings from role" selection) has an influence on the extent retrieved
+through the role and some functionalities.
+
+Role extents for users can only be set in one role, because the application
+is currently not able to check multiple extents for one user, thus it is the
+default role which defines this unique extent.
+
+Any functionality defined as ``single``, hence only one can be defined per
+user, has to be defined in this default role. Any ``single`` functionality
+defined in another role will not be taken into account.
+
+By default functionalities have ``single`` not set. Currently, the following
```

Comment:
Suggestion:
By default, functionalities are not specified as ``single``. Currently, ...